### PR TITLE
Edits to upgrade_server

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

### Description

Rewords sections, consistent punctuation.

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: content/upgrade_server.md [hunk 1]
```diff
@@ -153,90 +145,100 @@ For the latest information on high availability and how to set up a highly-avail
...
 
 ## Upgrading Add-ons
 
-This section details the process for upgrading additional features after the Chef Infra Server has been upgraded.
+Chef Infra Server 13 and 14 supports Chef Manage and push jobs. Both of these add-ons are deprecated and will reach [EOL](https://docs.chef.io/versions/#deprecated-products-and-versions) on December 31, 2021. After upgrading Chef Infra Server, reinstall the add-on and then reconfigure Chef Infra Server and the add-on.
+
+Chef Manage
+: Chef Manage is deprecated and will reach [EOL](https://docs.chef.io/versions/#deprecated-products-and-versions) on December 31, 2021. Chef Manage is a management console for  data bags, attributes, run-lists, roles, environments, and cookbooks from a web user interface
+
+Push jobs
```
Comment (IanMadd) [inline content/upgrade_server.md]:
```suggestion
Chef Push Jobs
```

File: content/upgrade_server.md [hunk 2]
```diff
@@ -153,90 +145,100 @@ For the latest information on high availability and how to set up a highly-avail
...
+: Push jobs deprecated and will reach [EOL](https://docs.chef.io/versions/#deprecated-products-and-versions) on December 31, 2021. Chef Push Jobs is an extension of the Chef Infra Server that allows for running jobs against nodes independently of a Chef Infra Client run.
```
Comment (IanMadd) [inline content/upgrade_server.md]:
```suggestion
: Chef Push Jobs is deprecated and will reach [EOL](https://docs.chef.io/versions/#deprecated-products-and-versions) on December 31, 2021. Chef Push Jobs is an extension of the Chef Infra Server will run jobs against nodes independently of a Chef Infra Client run.
```

File: content/upgrade_server_ha_v2.md [hunk 3]
```diff
@@ -145,49 +143,42 @@ cluster downtime.
...
 
     ```bash
     chef-server-ctl reconfigure
     ```
 
-8.  To continue the upgrades on Chef Infra Server frontend nodes using
-    this backend cluster, see [Upgrade Frontends Associated with a Chef
-    Backend
-    Cluster](/install_server_ha/#upgrading-chef-infra-server-on-the-frontend-machines)
+8. To continue the upgrades on Chef Infra Server frontend nodes using this backend cluster, see [Upgrade Frontends Associated with a Chef Backend Cluster](/install_server_ha/#upgrading-chef-infra-server-on-the-frontend-machines)
```
Comment (IanMadd) [inline content/upgrade_server_ha_v2.md]:
```suggestion
8. To continue the upgrades on Chef Infra Server frontend nodes using this backend cluster, see [Upgrade Frontends Associated with a Chef Backend Cluster](/install_server_ha/#upgrading-chef-infra-server-on-the-frontend-machines).
```

File: content/upgrade_server_ha_v2.md [hunk 4]
```diff
@@ -145,49 +143,42 @@ cluster downtime.
...
 ## DRBD/Keepalived HA to Chef Backend 2.x
 
 DRBD configurations are no longer supported. See [End of Life
 Products](/versions/#end-of-life-eol)
 
-For a guide to migrating to Chef Backend from DRBD see the [Best Best
-Practices for Migrating Your Chef
-Server](https://blog.chef.io/2018/04/06/best-practices-for-migrating-your-chef-server/)
+For a guide to migrating to Chef Backend from DRBD see the [Best Best Practices for Migrating Your Chef Server](https://blog.chef.io/2018/04/06/best-practices-for-migrating-your-chef-server/)
 webinar from the [Chef Blog](https://blog.chef.io/)
```
Comment (IanMadd) [inline content/upgrade_server_ha_v2.md]:
```suggestion
webinar from the [Chef Blog](https://blog.chef.io/).
```

File: themes/docs-new/layouts/shortcodes/ctl_chef_server_install_features_download.md [hunk 5]
```diff
@@ -1,31 +1,42 @@
...
+
+1. Install add-ons
+
+   Install Chef Manage with:
+
+   ```bash
+   sudo chef-server-ctl install chef-manage
+   ```
+
+   Install push jobs with:
```
Comment (IanMadd) [inline themes/docs-new/layouts/shortcodes/ctl_chef_server_install_features_download.md]:
```suggestion
   Install Chef Push Jobs with:
```

File: themes/docs-new/layouts/shortcodes/ctl_chef_server_install_features_download.md [hunk 6]
```diff
@@ -1,31 +1,42 @@
...
+
+1. Reconfigure add-ons
+
+   Reconfigure Chef Manage with:
+
+   ```bash
+   sudo chef-manage-ctl reconfigure
+   ```
+
+   Reconfigure push jobs with:
```
Comment (IanMadd) [inline themes/docs-new/layouts/shortcodes/ctl_chef_server_install_features_download.md]:
```suggestion
   Reconfigure Chef Push Jobs with:
```

File: themes/docs-new/layouts/shortcodes/ctl_chef_server_install_features_manual.md [hunk 7]
```diff
@@ -1,6 +1,4 @@
-The `install` subcommand is used to install premium features of the Chef
-server: Chef management console(`chef-manage`) and push
-jobs(`opscode-push-jobs-server`).
+Use the `install` subcommand with the `--path` option to install the Chef Manage (`chef-manage`) and push jobs(`opscode-push-jobs-server`) add-ons for Chef Infra Server.
```
Comment (IanMadd) [inline themes/docs-new/layouts/shortcodes/ctl_chef_server_install_features_manual.md]:
```suggestion
Use the `install` subcommand with the `--path` option to install the Chef Manage (`chef-manage`) and Push Jobs(`opscode-push-jobs-server`) add-ons for Chef Infra Server.
```

**Review by mjingle — APPROVED** [review]

File: content/upgrade_server.md (hunk 2 above)
Comment (kagarmoe) [reply content/upgrade_server.md]: It is just "push jobs", but I am unsure about the capitalization.

Comment (kagarmoe) [reply content/upgrade_server.md]: Went with "Push Jobs"

**Pull request merged**